exporters/zipkin/src/recordable.cc

```diff
@@ -215,6 +215,14 @@ void Recordable::SetDuration(std::chrono::nanoseconds duration) noexcept
   span_["duration"] = duration.count();
 }
 
+void Recordable::SetInstrumentationLibrary(
+    std::shared_ptr<const opentelemetry::sdk::instrumentationlibrary::InstrumentationLibrary>
+        &&instrumentation_library) noexcept
+{
+  span_["otel.library.name"]    = instrumentation_library->GetName();
+  span_["otel.library.version"] = instrumentation_library->GetVersion();
```

Comment:
As per the [specs ](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/zipkin.md#instrumentationlibrary)these should be reported in `tags`:

`OpenTelemetry Span's InstrumentationLibrary MUST be reported as tags to Zipkin using the following mapping.`

Similar to what is done for status:
https://github.com/open-telemetry/opentelemetry-cpp/blob/516d02d8bc19f778731d58c7162d16e08340207a/exporters/zipkin/src/recordable.cc#L194

Reply:
Thanks for the catch. I misread the `tags` part in the spec. Fixed it.